src/components/sidepanel/journeyDetails/Graph.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import {app} from "mobx-app";
+import {observer, inject} from "mobx-react";
+import styled from "styled-components";
+import doubleDigit from "../../../helpers/doubleDigit";
+import "../../../../node_modules/react-vis/dist/style.css";
```

Comment:
```suggestion
import "react-vis/dist/style.css";
```
should be sufficient.